docs/getting-started/requirements.md

```diff
@@ -71,14 +71,20 @@ To use each one you need to pass the proper flag to the execution. Prowler fro A
 
 #### Azure Active Directory scope
 
-Azure Active Directory (AAD) permissions required by the tool are the following:
+Microsoft Entra ID (AAD earlier) permissions required by the tool are the following:
 
 - `Directory.Read.All`
 - `Policy.Read.All`
 
 The best way to assign it is through the azure web console:
+1. Access to Microsoft Entra ID > App registrations and register an application
+  ![Register an Application page](../img/register-application.png)
+2. Once our application is registered, select *Api permissions* blade and add the needed permissions:
```

Comment:
```suggestion
2. Once your application is registered, select "API permissions" blade and add the needed permissions:
```

Reply:
What do you mean by "blade"?

Reply:
Done, a blade is like the item of the page that shows a new page